### PR TITLE
Edited

### PR DESCRIPTION
**Pull request by ghost**
- edited index and style.
- removed unnecessary svgs (?)

### CODE REVIEW

**Comment by Splinter0** [comment]
I really like the grid that you made for the map sizes, however you deleted svgs that were necessary to the page like : `discord.svg`, `level-1.svg`, `level-2.svg`, `level-3.svg`, `level-4.svg`. 
Check `index.html` and you'll see that they were necessary.
You removed the fading animation on the page... i don't know why you did that...
The font has to be the same as the one we had in the original, because the one you put is veeeery light.
So basically, keep all the same except for the table in the maps section, remove the svgs that are **actually** not needed and keep your name at the bottom.
@MineJoke @FrankWhoee

**Comment by Splinter0** [comment]
also @MineJoke keep the css optimization that you put, like putting classes with same attributes together.

**Comment by Splinter0** [comment]
oh and... you deleted other svgs which were needed and included in the `index.html`, like the season-II ceritficate.
Remember @MineJoke You can optimize the code, correct it and improve it. But we won't push such big design changes to master. :smile:

**Comment by Splinter0** [comment]
Closing because @MineJoke is gonna re-do it
